composables/useUser.ts

```diff
@@ -154,6 +155,47 @@ export const useUser = () => {
     }
   }
 
+  // updatedUserData should consist the user object clone along with updated values
+  const updateCustomerPersonalData = async (updatedUserData) => {
+    const variables = {
+      accountId: user.value.id,
+      customerAccountInput: updatedUserData,
+    }
+    try {
+      loading.value = true
+      const response = await fetcher({
+        query: updateCustomerDataMutation,
+        variables,
+      })
+      user.value = response.data?.updateCustomerAccount
+    } catch (err) {
+    } finally {
+      loading.value = false
+    }
+  }
+
+  const changePassword = async ({ oldPassword, newPassword }) => {
```

Comment:
Don't see test cases for covering updateCustomerPersonalData  and changePassword methods. Have we written it?